EIPS/eip-1.md

```diff
@@ -37,11 +37,31 @@ An EIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## EIP Work Flow
 
+### Shepherding an EIP
+
 Parties involved in the process are you, the champion or *EIP author*, the [*EIP editors*](#eip-editors), and the [*Ethereum Core Developers*](https://github.com/ethereum/pm).
 
-:warning: Before you begin, vet your idea, this will save you time. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [the Ethereum subreddit], [the Issues section of this repository], and [one of the Ethereum Gitter chat rooms]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EIP.
+It will be your role as the champion to socialize your EIP and build community consensus around it. Before you begin writing a formal EIP, you should vet your idea. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. It is recommended to open a discussion thread on [the Ethereum Magicians forum], but you can also use [one of the Ethereum Gitter chat rooms], [the Ethereum subreddit] or [the Issues section of this repository]. 
+
+In addition to making sure your idea is original, you should try and gauge whether the community interest in your EIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the community interest required for a technically complex Core EIP will have to be much higher than for a simple ERC. 
```

Comment:
I have been thinking over this the past week or so and finally distilled how I have been feeling on this. Up to this point EIPs have been touted as a purely technical process. This language gives more space to reject the submission of EIPs for non-technical reasons. I think that is fine, I just feel some clarifications would be beneficial.

The core of these two statements is that an Author has a duty to do due diligence around the EIP before submitting as a draft, and EIP Authors can reject an EIP because of lack of this due diligence. It would great to include this explicitly. 

Measuring the `community interest required` is difficult enough. Defending an EIP that has been refused because "the community interest required` hasn't been reached would be even more difficult. I am not sure we should create a "button" like this to be pressed. Unless the intent is to expand EIP editors role to refusing submitted EIPs based on community interest.

Reply:
I am focusing on the pre-submission stage, or pre 'merged as draft' stage. An EIP being rejected for the reason of community interest after it has reached the Draft stage is totally appropriate. If that switch is before it exists as draft, that puts a lot of burden on the EIP Editors to verify community sentiment. A little less ambiguity as to what happens when would be helpful.

Reply:
This is excellent feedback, @madeoftin, thank you very much!

To make sure I understand correctly, you are proposing to: 
1. Make it explicit that due diligence should be done pre-draft and not doing so is a valid reason for rejection;
2. Make it explicit that "community interest" is not sufficient to block an EIP from being merged as a draft, but that insufficient community interest may be grounds for rejection once the EIP has passed the Draft stage.

Re: (2), I think this is a particularly good idea. I feel that we may eventually need a way to signal that an EIP is a good idea, but not worth the time/resources/politics/etc. to be implemented. This is a first step in that direction, without being drastic given that it more or less reflects the current reality.

Reply:
@axic @nicksavers @carver given that you've approved the original version prior to this feedback, I'd love to get your thoughts on this as well. Thanks!

Reply:
@timbeiko I've thought about having expiry dates. Once an EIP before Final doesn't get updated by the author for a set period of time (let's say 6 months) - it's up to the editors prerogative to move the proposal to 'Inactive'.

From the IETF requirements
> A document expiration date should appear on the first and last page of the I-D.  The expiration date is 185 days following the I-D submission  of the document.  Use of the phrase "expires in six months" or "expires in 185 days" is not acceptable.

That might solve a lot of the problems around the lack of interest.

Reply:
Interesting! I think if we did want to do this, it would be out of scope from this PR, though. You would need a much stronger agreement from EIP maintainers & core devs than for this one, IMO, which only makes explicit a lot of what is already happening implicitly.

Reply:
> Measuring the `community interest required` is difficult enough. Defending an EIP that has been refused because `the community interest required` hasn't been reached would be even more difficult. I am not sure we should create a "button" like this to be pressed. Unless the intent is to expand EIP editors role to refusing submitted EIPs based on community interest.

Sure, and it's worthwhile to consider the difference between an Core EIP and an ERC for example. We need to get some sense of community support before a change is slated for a network upgrade, but I don't think community support should be required to approve EIPs for message signing formats, ABI standards (like ERC-20), etc.

Reply:
> Make it explicit that due diligence should be done pre-draft and not doing so is a valid reason for rejection;
> Make it explicit that "community interest" is not sufficient to block an EIP from being merged as a draft, but that insufficient community interest may be grounds for rejection once the EIP has passed the Draft stage.

Yes, this exactly. 

> and I've thought about having expiry dates. Once an EIP before Final doesn't get updated by the author for a set period of time (let's say 6 months) - it's up to the editors prerogative to move the proposal to 'Inactive'.

And yes this too please! It could also be automated so that the onus is on the authors to resurrect the proposal if they want to continue. That is bit more radical of a change, so a time period of inactivity being a requirement first is a better first step. I can go through and submit a PR to update once the criteria is decided.

Alex recently submitted an EIP to add an `Abandoned` status ethereum/eips#2051 Which fits in this context so linking that here.

Different words I have seen thrown around are:
 * Inactive
 * Withdrawn
 * Abandoned

Reply:
@carver good point. I will try to incorporate this as part of my second bullet above, which was:
> Make it explicit that "community interest" is not sufficient to block an EIP from being merged as a draft, but that insufficient community interest may be grounds for rejection once the EIP has passed the Draft stage.

Have a busy week ahead of me, but hopefully I can get an updated version of this out by AllCoreDevs to point people to it.